src/playwrightTestServer.ts

```diff
@@ -226,6 +226,23 @@ export class PlaywrightTestServer {
     let debugTestServer: TestServerConnection | undefined;
     const disposables: vscodeTypes.Disposable[] = [];
     try {
+      const debugEnd = new this._vscode.CancellationTokenSource();
+      token.onCancellationRequested(() => debugEnd.cancel());
+
+      let mainDebugRun: vscodeTypes.DebugSession | undefined;
+      this._vscode.debug.onDidStartDebugSession(session => {
+        if (session.name === 'Playwright Test')
```

Comment:
can't we use the `debugSessionName` constant here?
```suggestion
        if (session.name === debugSessionName)
```